gpii/node_modules/lifecycleManager/test/js/LifecycleManagerQueueTests.js

```diff
@@ -0,0 +1,128 @@
+/*!
+GPII Lifecycle Manager Queue Tests
+
+Copyright 2017 Raising The Floor - International
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+
+/* eslint-env browser */
+/* eslint strict: ["error", "function"] */
+
+/* global jqUnit, fluid */
+
+(function () {
+    "use strict";
+    fluid.setLogging(true);
+
+    var gpii = fluid.registerNamespace("gpii");
+
+    fluid.registerNamespace("gpii.tests.lifecycleManager.queue");
+
+    var promiseWrapFunc = function (timeout, func, reject) {
+        return function (arg) {
+            var myPromise = fluid.promise();
+            setTimeout(function () {
+                func(arg);
+                reject ? myPromise.reject(arg) : myPromise.resolve(arg);
+            }, timeout);
+            return myPromise;
+        };
+    };
+
+    var createCustomTask = function (timeout, func, arg) {
```

Comment:
Should explain clearly that it creates a object of the style which appears in the LM's queue. Also, this is a bit mixed since it creates the promise by itself, whereas in the impl the LM does it. Is line 41 necessary?